src/utils/systemIntake.ts

```diff
@@ -72,6 +73,29 @@ export const getAcronymForComponent = (componentToTranslate: string) => {
   return component ? component.acronym : 'Other';
 };
 
+/**
+ * Translate the API status enum to a human readable string
+ * @param statusEnum - intake status represented by enum value
+ */
+export const translateStatus = (
+  statusEnum: SystemIntakeStatus,
+  lcid: string | null
+) => {
+  let statusTranslation = '';
+
+  if (statusEnum === 'LCID_ISSUED') {
+    // if status is LCID_ISSUED, translate from enum to i18n and append LCID
+    statusTranslation = `${i18next.t(
+      `intake:statusMap.${statusEnum}`
+    )}: ${lcid}`;
```

Comment:
Should this have a case for handling a `null` LCID?

Reply:
Good idea,  it wont fail because it's a string | null but I don't think we want it to look like this:

![image](https://user-images.githubusercontent.com/68014929/155531088-25b7e577-0a84-4cb3-817e-6d31f8d9b8f0.png)

Thoughts on an alternative? We could just print out 'N/A' or 'LCID Not Available' like this:

![image](https://user-images.githubusercontent.com/68014929/155531410-758994ab-5346-4351-8196-c28b6990591e.png)

Or have a whole new i18n message like "LCID Issued but ... " (some explanation of what happened), not sure which approach would be best - might be worth bringing in Zoe

Reply:
I talked to Zoe, were gonna go with LCID Issued: ID Not Available